src/frontend/admin/src/components/Title.js

```diff
@@ -118,7 +118,17 @@ function TitleTableRow({ data, onShow, onEdit, onDelete }) {
 
     return (<TableRow key={data.id} sx={{ backgroundColor: (data.shown ? activeRowColor : undefined) }}>
         <TableCell component="th" scope="row" align={"left"} key="__show_btn_row__">
-            <ShowPresetButton onClick={onShow} active={data.shown}/>
+            <Box display="flex">
+                <ShowPresetButton className='block' onClick={onShow} active={data.shown}/>
+                <IconButton color="primary" onClick={() => {
+                    data.settings.preset = "title_left.svg";
```

Comment:
React doesn't like when props are mutated. Here you are mutating `data`. This is prone to bugs